io.go

```diff
@@ -1469,6 +1470,19 @@ func ReadGbkFlat(path string) []Sequence {
 	return sequences
 }
 
+// ReadGbkFlatGz reads flat gzip'd genbank files, like the ones provided by the NCBI FTP server
+func ReadGbkFlatGz(path string) []Sequence {
+	file, err := ioutil.ReadFile(path)
```

Comment:
If you're not using err for anything just replace it with an underscore.

Reply:
Fixed.